nixos/modules/virtualisation/oci-containers.nix

```diff
@@ -255,7 +255,9 @@ let
           ${cfg.backend} login \
           ${container.login.registry} \
           --username ${container.login.username} \
-          --password-stdin < ${container.login.passwordFile}
+          --password-stdin < ${container.login.passwordFile} \
+          || ${cfg.backend} image inspect ${container.image} >/dev/null \ # check if image exist locally
```

Comment:
I am somewhat sure that shellcheck would complain about this, as I think the bash ordering is not as we would expect it. Probably some more {} are required maybe after the first || but I am not sure.

Reply:
good catch! a `\` before a comment shouldn't work.
Consider moving the comment to above `${cfg.backend}login`

Reply:
thanks! updated